test/buy-tshirt.spec.ts

```diff
@@ -4,35 +4,57 @@ import {
   ShippingStep, PaymentStep, BankPayment, OrderSummary,
 } from '../src/page';
 
-describe('Buy a t-shirt', () => {
-  const menuContentPage: MenuContentPage = new MenuContentPage();
-  const productListPage: ProductListPage = new ProductListPage();
-  const productAddedModal: ProductAddedModal = new ProductAddedModal();
-  const summaryStep: SummaryStep = new SummaryStep();
-  const signInStep: SignInStep = new SignInStep();
-  const addressStep: AddressStep = new AddressStep();
-  const shippingStep: ShippingStep = new ShippingStep();
-  const paymentStep: PaymentStep = new PaymentStep();
-  const bankPayment: BankPayment = new BankPayment();
-  const orderSummary: OrderSummary = new OrderSummary();
-
-  it('then should be bought a t-shirt', async () => {
+describe('Go to Shopping page', () => {
+  beforeAll(async () => {
     browser.waitForAngularEnabled(false);
     await browser.get('http://automationpractice.com/');
-    await menuContentPage.goToTShirtMenu();
-    await productListPage.goToAddToCardMenu();
-    await productAddedModal.goToShoppingCartSummary();
-    await summaryStep.goToSignInStep();
+  });
+
+  describe('Buy a t-shirt process', () => {
+    const menuContentPage: MenuContentPage = new MenuContentPage();
+    const productListPage: ProductListPage = new ProductListPage();
+    const productAddedModal: ProductAddedModal = new ProductAddedModal();
+    const summaryStep: SummaryStep = new SummaryStep();
```

Comment:
you can move the initialization of page objects within the `beforeAll` of te corresponding describe